### PR TITLE
Atlas-edits fixups (fixed)

### PR DESCRIPTION
**Pull request by carlyc99**

### CODE REVIEW

File: web/atlas.json [hunk 1]
```diff
@@ -2586,7 +2586,6 @@
...
-{"id": "twvk4u", "submitted_by": "jpepo", "name": "Flag of Croatia", "description": "The flag of Croatia, a country at the crossroads of Central and Southeast Europe.", "website": "https://en.wikipedia.org/wiki/Flag_of_Croatia", "subreddit": "", "center": [98.5, 732.5], "path": [[72.5, 716.5], [72.5, 748.5], [124.5, 748.5], [124.5, 716.5], [72.5, 716.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:2589]: I don't see any duplicate entry for this.

**Review by AnonymousRandomPerson — CHANGES_REQUESTED** [review]
Address comments.

File: web/atlas.json (hunk 1 above)
Comment (carlyc99) [reply web/atlas.json:2589]: Message link https://discord.com/channels/960791635342524496/960982269579911188/960991116134608916 asked for a removal, but I can put it back in if you’d like.

Comment (AnonymousRandomPerson) [reply web/atlas.json:2589]: There was probably another duplicate beforehand that was cleaned up, so this entry can stay.

Comment (carlyc99) [reply web/atlas.json]: Added it back.

**Pull request merged**